spec/controllers/msa_components_controller_spec.rb

```diff
@@ -27,4 +27,31 @@
       expect(response).to have_http_status(:success)
     end
   end
+
+  describe "GET #edit" do
+     it "returns http success" do
+       compmgr_stub_auth
+       get :edit, params: { id: msa_component.id }
+       expect(response).to have_http_status(:success)
+     end
+   end
+
+  describe "PATCH #update" do
+    it "returns http success" do
```

Comment:
Shouldn't this description state that the call redirects correctly, rather than that it returns a success status?

Reply:
But #update is what I was referring to.